packages/cli/openapi-parser/src/v3/converters/operation/convertStreamingOperation.ts

```diff
@@ -31,10 +35,191 @@ export function convertStreamingOperation({
                 nonStreaming: undefined,
             };
         }
-        case "streamCondition":
-            break;
+        case "streamCondition": {
+            const streamingOperation = convertHttpOperation({
+                operationContext: {
+                    ...operationContext,
+                    operation: {
+                        ...operationContext.operation,
+                        requestBody: convertRequestBodyToInlinedRequest({
+                            context,
+                            requestBody: operationContext.operation.requestBody,
+                            streamCondition: streamingExtension,
+                            isStreaming: true,
+                        }),
+                    },
```

Comment:
you're probably already tracking but you'll have to also create a similar helper `createOperationResponse(streamingExtension.responseStream)`

Reply:
basically this helper will return 
```ts
{
  "200": {
    "application/json": {
        "schema": responseSchema
    }
  }
}
```

Reply:
Yup, tracking that but thanks for calling that out - need to port the implementation over from what we have in https://github.com/fern-api/fern/pull/2247